services/tasks-svc/main.go

```diff
@@ -14,12 +17,23 @@ const ServiceName = "tasks-svc"
 var Version string
 
 func main() {
+	ctx, cancel := context.WithCancel(context.Background())
+
 	common.Setup(ServiceName, Version, true)
 
 	eventStore := eventstoredb.SetupEventStoreByEnv()
 	aggregateStore := eventstoredb.NewAggregateStore(eventStore)
 
-	common.StartNewGRPCServer(common.ResolveAddrFromEnv(), func(server *daprd.Server) {
+	echoProjection := echo.NewProjection(eventStore)
+	go func() {
+		err := echoProjection.Base.Subscribe(ctx)
+		if err != nil {
+			log.Err(err).Msg("error during echo subscription")
+			cancel()
```

Comment:
cancel must probably also be called after StartNewGRPCServer has finished (see StartNewGRPCServer's comment)